### PR TITLE
Test all article list content

### PR DESCRIPTION
**Pull request by thewilkybarkid**
Makes sure that all quads in the article list are tested.

Found in #167.

### CODE REVIEW

File: test/routes/article-list.test.ts [hunk 1]
```diff
@@ -23,14 +23,35 @@ describe('article list', (): void => {
...
+  it('should contain articles', async (): Promise<void> => {
     const { dataset } = await makeRequest();
     const id = namedNode('http://example.com/path-to/article-list');
 
-    expect(dataset).toBeRdfDatasetContaining(quad(id, rdf.type, hydra.Collection));
-    expect(dataset).toBeRdfDatasetMatching({ subject: id, predicate: hydra.title });
     expect(dataset).toBeRdfDatasetMatching({ subject: id, predicate: hydra.manages });
-    expect(dataset).toBeRdfDatasetMatching({ subject: id, predicate: hydra.totalItems, object: toRdf(0) });
+
+    const [{ object: manages }] = dataset.match(id, hydra.manages);
```
Comment (giorgiosironi) [inline test/routes/article-list.test.ts:42]: if the predicate is `manages` (namespaces notwithstanding), is the object `manages` too? Maybe `managed`?

Comment (giorgiosironi) [reply test/routes/article-list.test.ts:42]: or just `article`?

Comment (giorgiosironi) [reply test/routes/article-list.test.ts:42]: assume if 2 articles suddenly come out the API, the `[{...}]` destructuring fails?

Comment (giorgiosironi) [reply test/routes/article-list.test.ts:42]: oh wait, this only describes the type of the items, not their content

Comment (thewilkybarkid) [reply test/routes/article-list.test.ts:42]: It's assigning an alias to a property on an object that is the first item in the array. Complicated syntax...

Destructuring is fine if there's more than one, others are ignored.

**Review by giorgiosironi — APPROVED** [review]

**Pull request merged**